app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/ui/onboarding/VpnOnboardingActivity.kt

```diff
@@ -306,12 +308,5 @@ class VpnOnboardingActivity : DuckDuckGoActivity() {
 
     companion object {
         private const val REQUEST_ASK_VPN_PERMISSION = 101
-
-        const val LAUNCH_FROM_NOTIFICATION_PIXEL_NAME = "LAUNCH_FROM_NOTIFICATION_PIXEL_NAME"
-
-        // TODO: ANA remove this.
-        fun intent(context: Context): Intent {
```

Comment:
This was re-added by me to solve the conflicts when rebased the `feature/ana/implementation_for_experiment_increase_retention_through_apptp_promotions` branch on `develop`.